.kokoro/tests/run_tests.sh

```diff
@@ -90,6 +90,9 @@ export PATH="${HOME}/.local/bin:${PATH}"
 # upgrade pip when needed
 pip install --upgrade pip
 
+# Install virtualenv in version 20.21, since version 20.22 no longer supports Python 2.7
+pip install --user -q virtualenv==20.21
+
```

Comment:
Can we split them for 2.7? There's a runtime specific condition above:
```
if [[ "${RUN_TESTS_SESSION}" == "py-3.8" ]]; then
```

Reply:
Sure, done!